test/e2e/scalability/density.go

```diff
@@ -819,7 +820,14 @@ func createRunningPodFromRC(wg *sync.WaitGroup, c clientset.Interface, name, ns,
 			},
 		},
 	}
-	_, err := c.Core().ReplicationControllers(ns).Create(rc)
+	var err error
+	for attempt := 0; attempt < MaxLatencyPodCreationTries; attempt++ {
```

Comment:
You're trying to create new RC. If you want to have retries, you'd need to delete RC created in previous run. I'm not sure it's a great idea.

Reply:
We're only going to retry in case of a retryable error which includes the following:

`apierrs.IsTimeout(err) || apierrs.IsServerTimeout(err) || apierrs.IsTooManyRequests(err) || apierrs.IsInternalError(err)`

I think in each of those cases the RC creation can be retried (with a slight doubt on the last one). And in case the rc has already been created, we're going to fail with a non-retryable error when retrying (which seems fine to fail on, because we would've failed anyway (even without retries) because the first try returned a non-nil error).

Reply:
Nope. If you got a Timeout you have no idea whether it succeeded or not. You only know that you haven't received a reply from the API server, but it may have been just eaten by dark gods of the internet. No way to know.

Reply:
I see. Though I'm not sure how that would hurt us in such cases where the resource already exists (as we were failing anyway, now we just retried and failed). Can you think of some better approach here?

Reply:
I guess you can check if Error is a collision, and if so, just assume it's created. As long as we won't be running multiple tests in the same time it should be fine.

Reply:
Hmm... I was thinking of checking something like 'AlreadyExists' error, but I'm not sure if that covers all the cases (for e.g what if there's sth else like Conflict?)

Reply:
Ok.. I've changed it to also check if it already exists for now. SG?